rir/src/compiler/opt/pass_definitions.h

```diff
@@ -133,6 +133,8 @@ class PASS(TypeInference);
 
 class PASS(TypeSpeculation);
 
+class PASS(HoistInstruction);
```

Comment:
Would you mind adding a comment specifying exactly what does this pass do?

Reply:
to have it become outdated like the others :)